version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.2.0"
+	Version = "1.0.0"
```

Comment:
What breaking changes does this change cause?

Also, are you planning to tag after merging this? If so, should this PR also contain a CHANGELOG update?

Reply:
At least, is switches Postgres version to 10 from 9.6. This is not done by myself, but I saw it and realized it is a breaking change.

CHANGELOG: +1, totally forgot that

Reply:
Wrote change log now. I will tag after this PR is merged.